jobs/config.json

```diff
@@ -3259,6 +3283,30 @@
       "sig-scalability"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-kube-dns": {
```

Comment:
All the comments from above job apply to here too.

Reply:
Also just to be consistent, you could remove the '-' b/w kube and dns.